add glif explorer (#1460)
filecoin-project/filecoin-docs: content/en/get-started/explore-the-network/index.md

weight: 7
---

## Account, actor & message explorers

> Account, actor & transaction explors allow you to view the states of a Fielcoin accounts and actors(smart contracts) and the details of the messages sent in the Filecoin network.

Reviewer: ```suggestion
These explorers allow you to view the states of Filecoin accounts and actors (smart contracts) and the details of messages sent across the network.
```

> ### Glif Explorer

Reviewer: ```suggestion
### Glif Explorer

```
